Python/Tests/ReplWindowUITests/ReplWindowSendUITests.cs

```diff
@@ -87,7 +94,7 @@ ...     print(y ** x)
         /// while it's executing.
         /// </summary>
         public void SendToInteractiveDelayed(PythonVisualStudioApp app) {
-            RunOne(app, "Delayed.py",
+            RunFromProject(app, "Delayed.py",
```

Comment:
How about adding "RunFromWorkspace" scenarios to these tests as well?

Reply:
I only added tests that triggers the differences in the product code when using a workspace. These other tests could be adapted for workspaces but wouldn't add to the code coverage (and would take up time to run).